llpc/util/llpcElfWriter.h

```diff
@@ -74,6 +74,22 @@ template <class Elf> class ElfWriter {
 
   static void updateMetaNote(Context *context, const ElfNote *note, ElfNote *newNote);
 
+  static size_t getRelocsCount(const SectionBuffer *pRelocSection);
+
+  static size_t getRelocPsStartPos(const SectionBuffer *pRelocSection, size_t psIsaOffset);
+
+  void createNewSection(const char *pSectionName, SectionBuffer *pInputSection, SectionBuffer *pNewSection);
```

Comment:
Why doesn't this return a pointer to the new section?
Also, can the section name argument be `StringRef`?

Reply:
I just follow other functions like `mergeSection` here, the new section is returned by argument.
Some of the section names are already defined by `const char *`， if change the type to `StringRef` that may cause additional cast.